### ISSUE TITLE
Can't run the example code

### ISSUE THREAD

**Issue by adeem** [steuerbot/react-native-idnow-autoident#1]
Thanks for open sourcing it. But when I try to run the example project, I m getting the below error. 
Even when I create an example project with react-native 0.68.2 and install this library using npm. I m getting the same issue as the below image. 

It possible, can you update your example project?

![image](https://user-images.githubusercontent.com/68442/192151243-dc9e3fcb-4121-414b-b414-452bae12dff6.png)

I even added the package by myself but it still gives an error 
![image](https://user-images.githubusercontent.com/68442/192182880-3d97c671-fb79-49b0-b2f2-e0b8a740787c.png)

**Comment by friedolinfoerder** [comment]
Yes, we struggled with the same and ended up integrating the SDK directly in the app project. Android is working but iOS not 😕

**Comment by thorgas** [comment]
> Yes, we struggled with the same and ended up integrating the SDK directly in the app project. Android is working but iOS not 😕

I got the Android version of this package running. Is it possible to add the direct integration for iOS into this repo via Screenshots, script or patch-package? Currently I have no idea how to do it.

**Comment by friedolinfoerder** [comment]
You can have a look at https://github.com/nickbuller/IDNowReactNative for the iOS implementation. This was basically the integration which works for us.
